### DISCUSSION TITLE
[feature request] How to stick a ipynb cell to avoid it scroll away?

### DISCUSSION THREAD

**Discussion by Roy-Kid** [microsoft/vscode-discussions#1216] [Extension Development QnA]
Hi everyone!

I find a feature: [can stick ](https://code.visualstudio.com/docs/datascience/jupyter-notebooks#_run-cells-in-section). Maybe there is an easy way to stick a cell and its output on the top?

The context is that I am developing an interactive molecule visualization package that can use Python API to control the view. I want to use it in jupyter. When I initialize a widget and it will display a canvas in cell's output, then I still can execute some commands to update the view. But after several cells, it will scroll update and you need to scroll back to see the result. Here is an extension named [stickyland](https://github.com/xiaohk/stickyland), but I can't find a similar one for vscode.  

<img width="1917" alt="695faa65b35bbec689ef095940e1eac" src="https://github.com/microsoft/vscode-discussions/assets/42854324/88f615a6-8388-45ec-9b51-0f040287ebc9">

I hereby request this feature. Or maybe I can make an extension? Can anyone provide an idea?
Thanks a lot for your kind help!

**Comment by isidorn** [comment]
This sounds like a feature request against VS Code. Please file it there https://github.com/microsoft/vscode/issues
This repo is for extension authors.
